src/utils/get-edge-path/get-edge-path.ts

```diff
@@ -11,8 +12,10 @@ export function getEdgePath(): string {
       return '%windir%/SystemApps/Microsoft.MicrosoftEdge_8wekyb3d8bbwe/MicrosoftEdge.exe';

```

Comment:
It seems to me that this install path targets the "old" Microsoft Edge. Now I think it makes sense to update this path to the 'chromium' Microsoft Edge.
I suggest, you also modify the `appveyor.yml` to force the installation of Microsoft edge so that you can get the path on windows machine.

It should be something like `???/Microsoft/Edge/Application/msedge.exe`.

Reply:
yes i will update